lua/autorun/client/cl_mcsp.lua

```diff
@@ -0,0 +1,98 @@
+local MCSPmodel = "models/milaco/minecraft_pm/minecraft_pm.mdl"
+CreateClientConVar("mcsp_url", "", true, true)
+
+local function MCSPRenderNewMaterial(url, ply) --This is where the magic happens, baby
+	ply.MCSPSkinMaterial = ImgurMaterial(url, ply, ply:GetPos(), false, "VertexLitGeneric", {
+		["$alphatest"] = 1,
+		["$halflambert"] = 1,
+		["$nodecal"] = 1,
+		["$model"] = 1
+	})
+end
+
+RegisterChatCommand({'mcsp','skinpicker'}, function(ply, arg)
```

Comment:
Chatcommands are serverside only.

What you probably want is a command like "!minecraftskin (url)"

What does "MCSP" mean? Using an internal name for your code like "sps" is fine but no player is gonna know it.

Reply:
it stands for Minecraft Skin Picker, because I was too lazy to type !minecraftskinpicker into chat when testing